### PR TITLE
add showfilename

### PR DESCRIPTION
**Pull request by phong9xNcc**

### CODE REVIEW

File: libs/transport/src/lib/minio/index.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export async function handleUploadFileMobile(
 
 export function createUploadFilePath(session: Session, currentClanId: string, currentChannelId: string, filename: string): string {
 	const ms = new Date().getMinutes();
-	filename = ms + filename;
```
Comment (nguyentrannhan) [inline libs/transport/src/lib/minio/index.ts:113]: cho nay truyen filename vao attachment

File: libs/components/src/lib/components/MessageWithUser/MessageLinkFile.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ function MessageLinkFile({ attachmentData }: MessageImage) {
 			<div className="flex items-center">{thumbnailAttachment}</div>
 			{hideTheInformationFile && (
 				<div className=" cursor-pointer " onClick={handleDownload} onKeyDown={handleDownload}>
-					<p className="text-blue-500 hover:underline">{attachmentData.filename}</p>
+					<p className="text-blue-500 hover:underline">{attachmentData.filename?.slice(attachmentData.filename?.lastIndexOf('/') + 1)}</p>
```
Comment (nguyentrannhan) [inline libs/components/src/lib/components/MessageWithUser/MessageLinkFile.tsx]: truyen lam 2 params rieng biet la filename voi url

File: libs/transport/src/lib/minio/index.ts (hunk 1 above)
Comment (phong9xNcc) [reply libs/transport/src/lib/minio/index.ts:113]: fixed:
1. exported filePath && originalFilename
image
2. set filename == original name
image

File: libs/components/src/lib/components/MessageWithUser/MessageLinkFile.tsx (hunk 2 above)
Comment (phong9xNcc) [reply libs/components/src/lib/components/MessageWithUser/MessageLinkFile.tsx]: image
removed detect filename form  fullfillname

**Pull request merged**